python/ray/train/examples/accelerate/accelerate_torch_trainer.py

```diff
@@ -19,6 +19,7 @@
     set_seed,
 )
 
+import ray.data
```

Comment:
Does the example not work without this?

Reply:
Ray Data has a lazy import defined [here](https://github.com/ray-project/ray/blob/f3c86d173994e23da21eb3f28c7ddca96fb79c6b/python/ray/__init__.py#L288-L294), but I've never really understood if this should be something done by all libraries, no libraries, or some libraries.

Reply:
I've tried that after we `import ray.train`, we can directly call anything methods of `ray.data`.

Reply:
Ray Data can be accessed after import ray directly. Examples [here](https://docs.ray.io/en/master/data/loading-data.html)

Reply:
oh that makes sense now. Ray train import ray, then ray.data is lazily imported.